lib/server/backends/sessions.ts

```diff
@@ -23,6 +23,7 @@ export function getStubSessionsBackend(): SessionsBackend {
         session = sessions[sid!]
         if (!exists(session)) {
           console.warn(`[IB] Missing or expired session '${sid}'`)
+          session = null
```

Comment:
`return null`

Reply:
Can't return here, need to create a new blank session below. Removed the null assignment however.